js/components/table/index.js

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
+import {repositories} from '../../mocks';
+import PropTypes from 'prop-types'
+import MyHighchart from '../highchart';
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+import styles from './table.scss';
+
+const i18n = {
```

Comment:
what the goal of this i18n object?

Reply:
I wanna create a localization for table(some dropdown for language on page)